RiverClan/urls.py

```diff
@@ -16,9 +16,11 @@
 from django.contrib import admin
 from django.urls import path, include
 from flash_cards import views
-
+from RiverClan import settings
```

Comment:
```suggestion
from RiverClan import settings

```
consider adding this line for clarity